esa/saw.py

```diff
@@ -791,9 +791,10 @@ def get_jacobian(self, full=False, Polar=False):
         jidfile = tempfile.NamedTemporaryFile(mode='w', delete=False)
         jidfile_path = Path(jidfile.name).as_posix()
         jidfile.close()
-        if Polar:
-            cmd = f'SaveJacobian("{jacfile_path}","{jidfile_path}",M,P);'
+        if (JacForm == "R") | (JacForm == "P") | (JacForm == "DC")::
```

Comment:
Actually you should add the docstring for the option `JacForm`, and then you dont need to add the validation logic in the code, cause everyone suppose to read the documentation before using the package

Reply:
Added the changes, only concern is if {JacForm} needs to have quotes around it or not "{JacForm}"

Reply:
no need to add the quotes